fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -1670,6 +1670,31 @@ public void replaceTempPartitions(List<String> partitionNames, List<String> temp
         }
     }
 
+    // used for unpartitioned table in insert overwrite
+    // replace partition with temp partition
+    public void replacePartition(String sourcePartitionName, String tempPartitionName) {
```

Comment:
It it strange to give partition names for unpartitioned

Reply:
unpartitioned table's partition also has a name, it is default to the table name. Here I have to add a tmp partition for table, so I need to give the tmp partition a name. I think it is ok.